client/src/Movies/SavedList.js

```diff
@@ -12,7 +13,9 @@ export default class SavedList extends Component {
         {this.props.list.map(movie => (
           <span className="saved-movie">{movie.title}</span>
```

Comment:
You could have made this into a `Link` component and set the `to` prop to point to `/movie/${movie.id}`